packages/ui/deno.json

```diff
@@ -11,6 +11,16 @@
   },
   "imports": {
     "@shoelace-style/shoelace": "npm:@shoelace-style/shoelace@^2.19.1",
-    "marked": "npm:marked@^4.0.0"
+    "marked": "npm:marked@^4.0.0",
+    "lit": "npm:lit@^3.0.0",
```

Comment:
this is super surprising - I thought we were using lit already for other v1 components

Reply:
I removed this to have it use the lit defined import from the higher level